kuma/wiki/constants.py

```diff
@@ -80,6 +80,10 @@
 ALLOWED_ATTRIBUTES['meter'] += ['max', 'min', 'value', 'low', 'high', 'optimum',
                                 'form']
 ALLOWED_ATTRIBUTES['details'] += ['open']
+# attributes required for codepen-iex
+ALLOWED_ATTRIBUTES['div'] += ['data-prefill', 'data-height', 'data-editable',
```

Comment:
It would be good to allow `data-theme-id` as well. At some point we should look into different themes for this including custom ones.